test/inspector/test-inspector-port-zero-cluster.js

```diff
@@ -17,14 +18,14 @@ if (cluster.isMaster) {
     }));
     worker.send('debugPort');
   }
-  process.on('exit', () => {
+  process.on('exit', common.mustCall(() => {
```

Comment:
IIUC, `common.mustCall()` on an exit handler won't do anything because the check for execution in `mustCall()` is itself dependent on the `exit` event. Any situation that would skip calling the function will also skip the check that the function has been called. If the event does not fire, then the check never happens. Unless I'm mistaken about all this, I'd say `common.mustCall()` should be removed because it provides a false guarantee that the function will be called.